docs/en/05_Contributing/01_Code.md

```diff
@@ -127,34 +127,30 @@ A core committer will also "label" your PR using the labels defined in GitHub, t
 
 The current GitHub labels are grouped into five sections:
 
-1. *Changes* - These are designed to signal what kind of change they are and how they fit into the [Semantic Versioning](http://semver.org/) schema
-2. *Impact* - What impact does this bug/issue/fix have, does it break a feature completely, is it just a side effect or is it trivial and not a bit problem (but a bit annoying)
-3. *Effort* - How much effort is required to fix this issue?
-4. *Type* - What aspect of the system the PR/issue covers
-5. *Feedback* - Are we waiting on feedback, if so who from? Typically used for issues that are likely to take a while to have feedback given
+1. *Impact* - What impact does this bug/issue/fix have, does it break a feature completely, is it just a side effect or is it trivial and not a bit problem (but a bit annoying)
```

Comment:
I still think it's worth mentioning here that this is evaluated in the context of CMS as a whole, not just for the specific module the issue is created for.

Reply:
Updated